style.css

```diff
@@ -0,0 +1,269 @@
+* {
+	box-sizing:	border-box;
+	font-family: "Arial";
+	font-size: 15px;
```

Comment:
in order for rem to work, you need to set the font size on the html element, rather than *